nautobot/extras/forms.py

```diff
@@ -187,24 +202,18 @@ class RelationshipFilterForm(BootstrapMixin, forms.Form):
 
     type = forms.MultipleChoiceField(choices=RelationshipTypeChoices, required=False, widget=StaticSelect2Multiple())
 
-    source_type = DynamicModelMultipleChoiceField(
-        queryset=ContentType.objects.all(),
+    source_type = MultipleContentTypeField(
```

Comment:
Oof there's this pattern again. We talked when I first started using it for the `Status` implementation about abstracting it to a custom field type that always orders by `("app_label", "model")`, but didn't do it. Please don't do it now, but perhaps we should consider slight refactor prior to v1.0.0-final.

Reply:
Sure, makes sense as a refinement of `MultipleContentTypeField` - I opened https://github.com/nautobot/nautobot/issues/98.